ThingIFSDK/ThingIFSDK/Command.swift

```diff
@@ -128,6 +125,20 @@ extension Command: FromJsonObject {
             aliasActionResults = []
         }
 
+        let commandState: CommandState?
+        if let state = jsonObject["commandState"] as? String {
+            commandState = CommandState(rawValue: state)
+        } else {
+            commandState = nil
+        }
+
+        let created: Date?
+        if let date = jsonObject["created"] as? Int64 {
+            created = Date(timeIntervalSince1970InMillis: date)
+        } else {
+            created = nil
```

Comment:
@rizakii 

This is required.
We must initialize `let` before using it. If we remove `else` statement, it cause build error.